Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.10.0
+version: 0.10.1
```

Comment:
I would prefer to bump the version number in another commit.

Especially this is a wrong version number bump according to https://semver.org/

Reply:
I will squash and merge any way for merge pr

Reply:
je dirai que c'est du patch de gateway donc c'est bon pou rla version

Reply:
no we had several new changes done by jean pommier last week, these are new features.
Please remove the version number bump in this PR I'll create another commit for that.